daemon/k8s_watcher.go

```diff
@@ -86,7 +86,7 @@ func k8sErrorHandler(e error) {
 		} else {
 			if strings.Contains(errstr, "Failed to list *v1.NetworkPolicy: the server could not find the requested resource") {
 				k8sErrMsgMU.Unlock()
-				log.Warn("Consider upgrading kubernetes to >=1.7 to enforce NetworkPolicy version 1")
+				log.Warn("Consider upgrading k8s to >=1.7 to enforce NetworkPolicy version 1")
```

Comment:
@aanm I'm still a bit confused by this error message. What are we trying to tell the user here?

Reply:
We only support API version v1, and that was introduced in 1.7. I think this message is there to explain that to the user when they use a too old version (I'm guessing about all this).

Reply:
I've filed #1873 to track this